iOS/MMIWG2S/Utilities/Network.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  Network.swift
+//  MMIWG2S
+//
+//  Created by Callum Johnston on 4/7/21.
+//  Copyright © 2021 Google LLC. All rights reserved.
+//
+
+import Foundation
+
+enum HttpContentType: String {
+    case json = "application/json"
+    case form = "application/x-www-form-urlencoded"
```

Comment:
At this point the server is not setup to support `x-www-form-urlencoded`. It supports a `multipart/form-data` for accepting the image file uploaded. This case needs to be added here. 

Adding files to a form request requires setting the `boundaries` and also the `MimeType`
Here is a snippet for handling `multipart/form-data` in `iOS`: [link](https://stackoverflow.com/a/64642247/1072825)

Reply:
Oh! Good to know, thanks @pavan4!